readme.md

```diff
@@ -232,15 +243,16 @@ wherever `belongs_to` appears in the model definition.
 
```

Comment:
Be prepared to talk about why we have both `belongs_to`  in the model and `t.references`. like how `belongs_to` sets up AR associations and `t.references` is adding a foreign key to the database